docs/streams/architecture.html

```diff
@@ -110,6 +110,11 @@ <h3 class="anchor-heading"><a id="streams_architecture_threads" class="anchor-li
         and Kafka Streams takes care of distributing partitions amongst tasks that run in the application instances. You can start as many threads of the application
         as there are input Kafka topic partitions so that, across all running instances of an application, every thread (or rather, the tasks it runs) has at least one input partition to process.
     </p>
+    <p>
+        As of Kafka 2.8 you may scale clients much in the same way you can scale your application.
```

Comment:
Shouldn't we use `stream threads` instead of `clients` here? We are actually scaling stream threads. It just happens to be that each stream thread has its own clients. This might change in future.

Reply:
You are right. We should also change application to KafkaStream clients